recipe-server/client/control/components/action_fields/OptOutStudyFields.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes as pt } from 'react';
+
+import { ControlField } from 'control/components/Fields';
+import ActionFields from 'control/components/action_fields/ActionFields';
+
+/**
+ * Form fields for the opt-out-study action.
+ */
+export default class OptOutStudyFields extends ActionFields {
```

Comment:
At this point it's more just there to document the expected fields on the class, even if they don't inherit, sort've like an interface. I also suspected that we'd add specific, inheritable methods to the class in the future and figured using a base class now wouldn't hurt.